lib/rrd.ml

```diff
@@ -641,11 +629,11 @@ let xml_to_output rrd output =
   let do_ds ds output =
```

Comment:
This code suggests something like:
```
let emit label value = tag label (data value) output in
emit "name"  (data ds.ds_name);
emit "type"  (ds_type_to_string ds.ds_ty);
...

```

Reply:
I think that if `output` is mutable the code can be further simplified, don't really understand why it is passed around so much. `Xmlm.output` is deprecated too.